hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -173,6 +173,74 @@ def hpat_pandas_series_shape_impl(self):
     return hpat_pandas_series_shape_impl
 
 
+@overload_method(SeriesType, 'std')
+def hpat_pandas_series_std(self, axis=None, skipna=None, level=None, ddof=1, numeric_only=None):
+    """
+    Pandas Series method :meth:`pandas.Series.std` implementation.
+
+    .. only:: developer
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_std
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_std_unboxing
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_std_str
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_std_unsupported_params
+
+    Parameters
+    ----------
+    self: :obj:`pandas.Series`
+        input series
+    axis: :obj:`int`, :obj:`str`
+        Axis along which the operation acts
+        0/None/'index' - row-wise operation
+        1/'columns'    - column-wise operation
+        *unsupported*
+    skipna: :obj:`bool`
```

Comment:
skipna : bool, default True

Reply:
But default value is `None` according to Pandas documentation on this method.